.ameba.yml

```diff
@@ -0,0 +1,10 @@
+# This configuration file was generated by `ameba --gen-config`
+# on 2020-04-27 21:05:58 UTC using Ameba version 0.12.0.
+# The point is for the user to remove these configuration records
+# one by one as the reported problems are removed from the code base.
+
+# Problems found: 2
+# Run `ameba --only Style/ConstantNames` for details
```

Comment:
Think we could fix this?

Reply:
You mean so we don't have to have this disabled? This catches `Log` and says it should be `LOG`, but that would require everyone updating their apps to be `LOG.info {}`, etc... which doesn't really follow the crystal way. I think there was another one with one of the Pipes that it wanted PIPES...

Reply:
Ah yeah, that's a bummer. Thanks for the explanation. I agree with you keeping it the way it is despite the custom config.

Reply:
I removed a comment on how many errors it found. It was irrelevant. I do like this comment that tells us how we can run to see those errors though if we ever need.

Reply:
I think @veelenga was thinking of changing this in the future so we don’t need the comment

Reply:
Hey, thanks for mentioning. This is already changed in ameba [`v0.12.1`](https://github.com/crystal-ameba/ameba/releases/tag/v0.12.1). You can safely upgrade.